packages/web/scripts/workers-site/index.js

```diff
@@ -128,7 +130,17 @@ function clean(str) {
     .replace(/"/g, '&quot;')
 }
 
-class HeadElementHandler {
+class SEOHandlerBody {
+  element(element) {
+    if (!h1) {
+      return
+    }
+    const h1Tag = `<h1 id="audius-h1" style="visibility:hidden;display:none;">${h1}</h1>`
```

Comment:
so visibility hidden and display dont matter from seo perspective? i just know some systems ignore elements with those tags